packages/lwc-language-server/src/javascript/__tests__/compiler.test.ts

```diff
@@ -103,74 +34,35 @@ export default class Foo extends LightningElement {
 }
 `;
 
-it('transform throws exceptions on syntax errors', async () => {
-    try {
-        const options: CompilerOptions = {
-            name: 'foo',
-            namespace: 'x',
-            files: {},
-        };
-        await transform(codeSyntaxError, 'foo.js', options);
-        fail('expects exception');
-    } catch (err) {
-        // verify err has the info we need
-        const message = extractMessageFromBabelError(err.message);
-        expect(message).toMatch('Unexpected token (4:17)');
-        expect(err.location).toEqual({ line: 4, column: 17 });
-    }
+it('can get metadata from a simple component', async () => {
+    await compileSource(codeOk, 'foo.js');
 });
```

Comment:
We've added a test case for a bare-bones LWC component here. This does little more than ensure that the plumbing works and that errors are not thrown.